### PR TITLE
feature: Add translations for different languages

### LINKED ISSUE #3019: Add more translations
Ensure that Avo is translated into all of these languages, besides the ones that we already support:

- [x] - german
- [ ] - spanish
- [x] - russian
- [ ] - french
- [ ] - japanese
- [x] - chinese
- [x] - italian
- [x] - polish
- [x] - dutch
- [x] - ukranian

Also see if we can set up this strategy from [Superails](https://superails.com) to keep the translations in sync
https://superails.com/posts/rails-173-ai-automated-translations?playlist=i18n-internationalization

### PR DESCRIPTION
**Pull request by binarygit**
- Add German Translations
- Add dutch translations
- Add russian translations
- Add chineze translations
- Add italian translations
- Add polish translations
- Add ukranian translations

# Description

Fixes #3019

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the [documentation](https://github.com/avo-hq/avodocs)
- [ ] I have added tests that prove my fix is effective or that my feature works

## Manual review steps

1. Step 1
1. Step 2

Manual reviewer: please leave a comment with output from the test if that's the case.

### CODE REVIEW

File: lib/generators/avo/templates/locales/avo.de.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+de:
```
Comment (adrianthedev) [inline lib/generators/avo/templates/locales/avo.de.yml:1]:
```suggestion
---
de:
```

File: lib/generators/avo/templates/locales/avo.uk.yml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  avo:
+    resource_translations:
+      product:
+        save: "Зберегти продукт!"
+    action_ran_successfully: Дія успішно виконана!
+    actions: Дії
+    and_x_other_resources: і ще %{count} інших ресурсів
+    applied: застосовано
+    are_you_sure: Ви впевнені?
+    are_you_sure_detach_item: Ви впевнені, що хочете відокремити цей %{item}?
```
Comment (yshmarov) [inline lib/generators/avo/templates/locales/avo.uk.yml:12]: use `відкріпити`, not `відокремити` everywhere
use `відкріплено`, not `відокремлено` everywhere

```suggestion
    are_you_sure_detach_item: Ви впевнені, що хочете відкріпити цей %{item}?
```

File: lib/generators/avo/templates/locales/avo.uk.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    attachment_class_attached: "%{attachment_class} прикріплено."
+    attachment_class_detached: "%{attachment_class} відокремлено."
+    attachment_destroyed: Вкладення знищено
+    cancel: Скасувати
+    choose_a_country: Виберіть країну
+    choose_an_option: Виберіть опцію
+    choose_item: Виберіть %{item}
+    clear_value: Очистити значення
+    click_to_reveal_filters: Натисніть, щоб показати фільтри
+    close_modal: Закрити модальне вікно
```
Comment (yshmarov) [inline lib/generators/avo/templates/locales/avo.uk.yml:26]:
```suggestion
    close_modal: Закрити вікно
```

File: lib/generators/avo/templates/locales/avo.uk.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+    failed: Не вдалося
+    failed_to_find_attachment: Не вдалося знайти вкладення
+    failed_to_load: Не вдалося завантажити
+    file:
+      one: файл
+      other: файлів
+    filter_by: Фільтрувати за
+    filters: Фільтри
+    go_back: Назад
+    grid_view: Сітковий вигляд
```
Comment (yshmarov) [inline lib/generators/avo/templates/locales/avo.uk.yml:52]:
```suggestion
    grid_view: Плитка
```


that's the word used here https://rozetka.com.ua/ua/routers/c80193/

File: lib/generators/avo/templates/locales/avo.uk.yml [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    search:
+      cancel_button: Скасувати
+      placeholder: Пошук
+    select_all: Вибрати все
+    select_all_matching: Вибрати всі відповідні
+    select_item: Вибрати елемент
+    show_content: Показати вміст
+    sign_out: Вийти
+    switch_to_view: Перейти до виду %{view_type}
+    table_view: Табличний вигляд
```
Comment (yshmarov) [inline lib/generators/avo/templates/locales/avo.uk.yml:103]:
```suggestion
    table_view: Таблиця
```

File: lib/generators/avo/templates/locales/avo.uk.yml [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+    close_modal: Закрити модальне вікно
+    confirm: Підтвердити
+    create_new_item: Створити новий %{item}
+    dashboard: Панель керування
+    dashboards: Панелі керування
+    default_scope: Всі
+    delete: Видалити
+    delete_file: Видалити файл
+    delete_item: Видалити %{item}
+    detach_item: Відокремити %{item}
```
Comment (adrianthedev) [inline lib/generators/avo/templates/locales/avo.uk.yml:35]:
```suggestion
    detach_item: Відкріпити %{item}
```

**Pull request merged**